CMakeLists.txt

```diff
@@ -26,6 +26,14 @@ set(CMAKE_CXX_STANDARD_REQUIRED on)
 
 project(umoria)
 
+#
+# Get around the fact that Visual Studio doesn't have ssize_t
+# defined but uses SSIZE_T instead.
+#
+if(MSVC)
+    add_definitions(-Dssize_t=SSIZE_T)
+endif(MSVC)
```

Comment:
Does this need to be here at the top of the config? Is it not possible to have it at the bottom of the file? Maybe here: https://github.com/dungeons-of-moria/umoria/blob/master/CMakeLists.txt#L240

Reply:
I tend to put those at the top of the file to make it clear that it applies to all things that are added, but it looks like it works just as well down below.  I've changed this in a commit on that branch.

With this, it compiles out of the box; you have to point CURSES_CURSES_LIBRARY at the pdcurses.lib and CURSES_INCLUDE_PATH one directory above where curses.h lives.

To run it, you need to copy the scores.dat file into the location of the executable (which on Windows is a subdirectory under the build directory named based on the build).

Reply:
Also the data directory needs to be copied there.

Reply:
Thank you for keeping umoria alive!  It was a blast from the past to get to play it again.

Reply:
Great, thanks for the extra info. I will definitely take a look at building the Windows versions with PDCurse.